src/main/scala/Core/core.scala

```diff
@@ -79,13 +79,14 @@ class CoreIO(implicit val conf: FlexpretConfiguration) extends Bundle {
   val host = new HostIO()
   val gpio = new GPIO()
   val int_exts = Input(Vec(8, Bool()))
+
+  val thread_id = Output(UInt(conf.threadBits.W))
```

Comment:
Is this a debug signal or is it used for something?

Reply:
Looks like I forgot to remove it - nice catch!